unitTests/ui/QuerySuggestPreviewTest.ts

```diff
@@ -194,6 +234,22 @@ export function QuerySuggestPreviewTest() {
         expect(test.cmp.queryController.getLastQuery().q).toBe(realQuery);
         done();
       });
+
+      it(`and the query get executed, 
+      an analytics get logs`, () => {
```

Comment:
For the second line of the description, I would suggest `it logs an analytics search event`.